src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -906,6 +913,42 @@ private static bool AddFlags(TypeSymbol type, ArrayBuilder<bool> transformFlagsB
 
                 // Continue walking types
                 return false;
+
+                static void handleFunctionPointerType(FunctionPointerTypeSymbol funcPtr, ArrayBuilder<bool> transformFlagsBuilder, bool isNestedNamedType, bool addCustomModifierFlags)
+                {
+                    Func<TypeSymbol, (ArrayBuilder<bool>, bool), bool, bool> visitor = addFlags;
+
+                    transformFlagsBuilder.Add(false);
+
+                    var sig = funcPtr.Signature;
+                    handle(sig.RefKind, sig.RefCustomModifiers, sig.ReturnTypeWithAnnotations);
+
+                    foreach (var param in sig.Parameters)
+                    {
+                        handle(param.RefKind, param.RefCustomModifiers, param.TypeWithAnnotations);
+                    }
+
+                    void handle(RefKind refKind, ImmutableArray<CustomModifier> customModifiers, TypeWithAnnotations twa)
+                    {
+                        if (refKind != RefKind.None)
+                        {
+                            transformFlagsBuilder.Add(false);
+                        }
+
+                        if (addCustomModifierFlags)
+                        {
+                            HandleCustomModifiers(customModifiers.Length, transformFlagsBuilder);
```

Comment:
>customModifiers [](start = 50, length = 15)

These come before the ref. Even though, given the current implementation of HandleCustomModifiers the result is the same, consider handling things in metadata order.